benchmark/Benchmark.h

```diff
@@ -40,58 +42,113 @@ class BenchmarkRecords {
       // The names of the columns and rows.
       std::vector<std::string> rowNames;
       std::vector<std::string> columnNames;
-      // The actual content of the fields. As in the seconds one function
-      // execution takes. Access is [row, column].
-      std::vector<std::vector<float>> entries;
+      // The time measurements in seconds. Access is [row, column].
+      std::vector<std::vector<std::optional<float>>> entries;
+
+      // Default constructor. Without it, some of the included headers generate
+      // errors.
+      RecordTable(): descriptor{""}, rowNames{{}}, columnNames{{}}, entries{{}} {}
+
+      RecordTable(const std::string& pDescriptor, 
+          const std::vector<std::string>& pRowNames,
+          const std::vector<std::string>& pColumnNames):
+          descriptor{pDescriptor}, rowNames{pRowNames},
+          columnNames{pColumnNames},
+          entries(rowNames.size(), std::vector<std::optional<float>>(rowNames.size())) {}
```

Comment:
I'm gonna try and see, if this works. Because initialization with `{}` has the bad habit of choosing the wrong constructors, I tend to use `()`, when I want to use a specific constructor.

Reply:
Once again, `{}` seems to be choosing the wrong constructors.

If you compile with it, after changing the `{}` to `()`, the UnsortedAndSorted... benchmark generates segmentation faults with any option choosen. That is not the case, if you keep the `()`.